README.md

```diff
@@ -0,0 +1,127 @@
+
+```yaml
+- name: Add user olimex to i2c group
+  user:
+    name: olimex
+    groups: i2c
+    append: true
+```
+
+As can bee seen, tasks can be given a human-readable name.
+During execution of the playbook, Ansible will print out which tasks it executed, and what the status of the task is, e.g. whether it was successful, it changed something on the host or maybe it failed.
+
+### Handlers
+
+Handlers are similar to tasks, but they are only triggered when notified by other tasks.
+They are typically used to restart services or perform specific actions after a change has been made on the target hosts.
+Handlers ensure that actions are performed only when necessary.
+By default, the notified handlers run after all the tasks in a particular play have been completed. 
+
+### Variables
+
+Ansible allows you to define [variables](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#playbooks-variables) to customize the behavior of your playbooks and tasks.
+Variables can be set globally in inventory files, defined in playbooks, or sourced from external files.
+They provide flexibility and allow you to reuse playbooks across different environments.
+
+One very common use-case for variables is in [templates for configuration files](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_templating.html).
+
+The following example tasks adds the user specified by the (built-in) variable `ansible_user` to the group `i2c`.
+```yaml
+- name: Add the Ansible connection user to the i2c group.
+  user:
+    name: '{{ ansible_user }}'
+    groups: i2c
+    append: true
+```
+
+### Modules
+
+Ansible comes with a wide range of [built-in modules](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html) that perform specific actions on target hosts.
+Modules are used within tasks to carry out operations like managing files, executing commands, manipulating data, or interacting with cloud providers.
+Modules can be executed remotely over SSH or leverage connection plugins for different protocols.
+
+[Ansible Galaxy](https://galaxy.ansible.com/) is a hub for sharing and discovering collections of moudles created by the Ansible community.
+It provides a centralized repository where you can find pre-written solutions for common tasks, saving you time and effort.
+
+## How to use this Repository
+
+This repository contains playbooks for the most basic tasks when setting up your OLinuXino within the [playbooks](playbooks/) directory.
+
+- [playbook.yaml](playbooks/playbook.yaml):
+  The container playbook that includes all the other playbooks.
+  
+- [bootstrap.yaml](playbooks/bootstrap.yaml): 
+  For most tasks, Ansible requires Python to be installed.
+  This playbook uses [raw commands](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/raw_module.html) to check for the presence of Python, and if it isn't found it will install a minimal Python environment on the host.
```

Comment:
This explains what bootstrap.yaml does, not how a user could use bootstrap.yaml. So this probably should be moved in another section.